java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -1239,48 +1281,48 @@ public Builder appendUTF8String(byte[] value, int offset, int length) {
 
     public Builder appendArray(byte... values) {
       assert (values.length + currentIndex) <= rows;
-      assert type == DType.INT8 || type == DType.UINT8 || type == DType.BOOL8;
-      data.setBytes(currentIndex * type.sizeInBytes, values, 0, values.length);
+      assert type.typeId == DType.INT8 || type.typeId == DType.UINT8 || type.typeId == DType.BOOL8;
```

Comment:
Now that we are making a change this would be a good time to clean this up and have a helper method in DataType like the one we have e.g. 'isTimestamp` etc

Reply:
Done